src/PrestaShopBundle/Resources/views/Admin/Sell/Catalog/Product/Combination/paginated_list.html.twig

```diff
@@ -36,8 +35,26 @@
         <div class="spinner spinner-primary"></div>
       </div>
 
-      {# Integrate the token that will be required in the combination update request #}
-      {{ form_row(combinationsForm, {'attr': {'data-combination-token': csrf_token(combinationItemForm.vars.name) }}) }}
+      <div id="combinations-list-form-container">
+        {{ include('@PrestaShop/Admin/Sell/Catalog/Product/Combination/combination_list_form.html.twig', {'combinationsForm': combinationsForm}) }}
+      </div>
+
+      <div id="combinations-list-footer">
+        <span>
+          {{ 'Edition mode'|trans({}, 'Admin.Actions') }}
+        </span>
+        <div id="combinations-list-footer-buttons">
+          <button type="button" id="cancel-combinations-edition" class="btn btn-secondary">
+            {{ 'Cancel'|trans({}, 'Admin.Actions') }}
+          </button>
+          <button type="button" id="reset-combinations-edition" class="btn btn-secondary">
+            {{ 'Reset values'|trans({}, 'Admin.Actions') }}
+          </button>
+          <button type="button" id="save-combinations-edition" class="btn btn-primary">
+            {{ 'Save list'|trans({}, 'Admin.Catalog.Feature') }}
```

Comment:
Couldn't we use "Save" only?

Reply:
I let you judge if it's relevant, but for me it makes sense to have a slightly different wording here. Because on edition mode we have two buttons:
- the button to save the combination list independently via an ajax query
- the button in the footer to save the product form

Both buttons have very different actions so I think the idea was to emphasize this by using two different wordings.

Reply:
Ok, thanks for the explanation!